server/lessons/el/prosarmosmenes-eidopoihseis.html

```diff
@@ -60,8 +60,8 @@ <h2>Εδώ τελειώνει το μάθημα!</h2>
   <div class="next">
     <h4>Συνεχίστε τα μαθήματα:</h4>
     <ol>
-      <li><a href="/lessons/el/syllogh-pinaka/">Συλλογή και παρακολούθηση πίνακα από ιστοσελίδα</a></li>
-      <li><a href="/lessons/el/kathariste-vromika-dedomena/">Συλλέξτε περιεχόμενο από μια ιστοσελίδα και παρακολουθήστε το</a></li>
+      <li><a href="syllogh-pinaka">Συλλογή και παρακολούθηση πίνακα από ιστοσελίδα</a></li>
+      <li><a href="kathariste-vromika-dedomena">Συλλέξτε περιεχόμενο από μια ιστοσελίδα και παρακολουθήστε το</a></li>
```

Comment:
The text in `<a href="kathariste-vromika-dedomena"></a>` should be "Καθαρίστε και τυποποιήστε δεδομένα"